src/bastion-executor/benches/blocking.rs

```diff
@@ -15,19 +12,17 @@ use test::Bencher;
 #[bench]
 fn blocking(b: &mut Bencher) {
     b.iter(|| {
-        let handles = (0..10_000)
+        (0..100)
```

Comment:
Why is this decreased to 100?

Reply:
oh it shouldn't anymore, will put it back!